.changeset/curvy-ghosts-talk.md

```diff
@@ -0,0 +1,18 @@
+---
+"@logto/client": patch
+"@logto/browser": patch
+"@logto/capacitor": patch
+"@logto/chrome-extension": patch
+"@logto/express": patch
+"@logto/next": patch
+"@logto/node": patch
+"@logto/nuxt": patch
+"@logto/react": patch
+"@logto/remix": patch
+"@logto/sveltekit": patch
+"@logto/vue": patch
```

Comment:
you can bump client only. other packages will be bumped automatically

Reply:
ok, let me try